BitwardenShared/UI/Platform/Settings/Settings/AutoFill/AutoFillView.swift

```diff
@@ -14,19 +14,42 @@ struct AutoFillView: View {
 
     var body: some View {
         VStack(spacing: 20) {
+            autofillActionCard
+
             autoFillSection
 
             additionalOptionsSection
         }
+        .animation(.easeInOut, value: store.state.badgeState?.autofillSetupProgress == .complete)
```

Comment:
super small nit, but should this logic here be done as a computed property in the state?

Reply:
This is pretty simple, is there advantage of moving it to the state? This vs exposing a `isAutofillSetupProgressComplete` variable would be similar?

Reply:
no real advantage and i totally agree is pretty simple.  I guess I was looking to far into the pattern of removing any business logic from the view so its "dumb"

Reply:
Makes sense, I agree with that in general though!